packages/xstate-react/test/useSelector.test.tsx

```diff
@@ -303,8 +304,8 @@ describeEachReactMode('useSelector (%s)', ({ suiteKey, render }) => {
     });
 
     const identitySelector = (value: any) => value;
-    const getSnapshot = (actor: ReturnType<typeof createActor>) =>
-      actor.latestValue;
+    const getSnapshot = (actor: ActorRef<any, any>) =>
+      (actor as any).latestValue;
```

Comment:
Why is `as any` needed here (and below)?

Reply:
It's because this `.latestValue` is not actually part of the `ActorRef<any, any>`. `useSelector` actually works OK with such a thing but the problem is that this test grabs the reference to this actor from `state.context.childActor` and that is typed as `ActorRef<any, any>` and not as "`CustomActor`". So I really had to introduce casting **somewhere** here - and it made sense to me to just use it here. Ultimately, it doesn't really matter all that much